.devcontainer/devcontainer.json

```
@@ -1,16 +1,19 @@
 {
     "name": "devcontainer",
     "image": "ghcr.io/kubefirst/devcontainers/full",
+    "forwardPorts": [
+        3000
+    ],
     "features": {
-      "ghcr.io/shinepukur/devcontainer-features/vale:1": {}
+        "ghcr.io/shinepukur/devcontainer-features/vale:1": {}
     },
     "customizations": {
-      "vscode": {
-        "extensions": [
-          "unifiedjs.vscode-mdx",
-          "ChrisChinchilla.vale-vscode"
-        ],
-        "settings": {}
-      }
+        "vscode": {
+            "extensions": [
+                "unifiedjs.vscode-mdx",
+                "ChrisChinchilla.vale-vscode"
+            ],
+            "settings": {}
+        }
     }
-}
+}
```

```suggestion
}

```

Please add a trailing line - should be solved by EditorConfig plugin

---

Fixed this with the included .editorconfig file (last commit)